### ISSUE TITLE
apply, write etc missing on STM32F072x

### ISSUE THREAD

**Issue by Sickeroni** [kvasir-io/Kvasir#81]
apply, write set etc don't work. 
works on MKL27Z4 trough generated "Chip/MKL27Z4.hpp" thats include generated "Io.hpp" in that includes "Register/Register.hpp". Io.hpp is missing on STM32F072x and therefore Registers.hpp. 

helperscript must be modified for this.

**Comment by odinthenerd** [comment]
yes this is a problem, I think its part of a bigger question: "how best to make a split header implementation?" the way we are doing it now is not strictly split header because we include different named headers. It would be good if someone could, at the command line, decide which target to build.

**Comment by odinthenerd** [comment]
work around for now is to just include Register.hpp

**Comment by odinthenerd** [comment]
We should put the "include all" files in the same folder as the rest of the chip files and call it all.hpp, then it should include Register.hpp too.

**Comment by odinthenerd** [comment]
Chip should not be a sub folder of lib because lib is an include directory and this confuses intelisense because it walks all sub folders
